### PR TITLE
chore(hm-instances): Add page about available HM instances

### PR DESCRIPTION
**Pull request by MaciejTe**
chore(hm-instances): Add page about available HM instances

Changelog: None
Ticket: MEN-5970
Signed-off-by: Maciej Tomczuk <maciej.tomczuk@northern.tech>

# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [ ] Make sure that all commits follow the conventional commit [specification](https://www.github.com/mendersoftware/mendertesting/commitlint/grammar.md) for the Mender project.

The majority of our contributions are fixes, which means your commit should have
the form below:

```
fix: <SHORT DESCRIPTION OF FIX>

<OPTIONAL LONGER DESCRIPTION>

Changelog: <USER-FRIENDLY-CHANGE-DESCRIPTION> or <None>
Ticket: <TICKET NUMBER> or <None>
```

- [ ] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

File: 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md [hunk 1]
```diff
@@ -181,6 +181,8 @@ Keep this terminal open as we will shortly use it to install the Mender client.
 
 ## Step 3 - Login to hosted Mender
 
+!!! Mender provides various [regions](../../../10.General/00.Hosted%20Mender%20regions/docs.md) to connect to. Make sure you connect to the right instance.
```
Comment (tranchitella) [inline 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md:184]:
```
Mender is available in multiple regions. Make sure you select your desired one before proceeding.
```

File: 10.General/00.Hosted Mender regions/docs.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Overview
+taxonomy:
+    category: docs
+---
+
+Hosted Mender provides various domains to connect to, depending on what are your data processing and compliance needs.
```
Comment (tranchitella) [inline 10.General/00.Hosted Mender regions/docs.md:7]:
```suggestion
Mender is available in multiple regions you can choose from, depending on your data processing and compliance needs. Each instance is independent of the others and provides all the Mender features.
```

File: 10.General/00.Hosted Mender regions/docs.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+
+## Regions
+
+Currently Mender has two instances:
```
Comment (tranchitella) [inline 10.General/00.Hosted Mender regions/docs.md:12]:
```suggestion
The list of the currently supported regions follows:
```

File: 10.General/00.Hosted Mender regions/docs.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+* US instance - `https://hosted.mender.io`
+* EU instance - `https://eu.hosted.mender.io` 
+
+EU instance is compliant with General Data Protection Regulation,(GDPR).
```
Comment (tranchitella) [inline 10.General/00.Hosted Mender regions/docs.md:17]:
```suggestion
The EU instance is compliant with General Data Protection Regulation (GDPR).
```

File: 10.General/00.Hosted Mender regions/docs.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+Both instances are fully independent.
```
Comment (tranchitella) [inline 10.General/00.Hosted Mender regions/docs.md:19]:
```suggestion
```

File: 10.General/00.Hosted Mender regions/docs.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+## Region selection
+
+When setting up your account, you can choose between available Hosted Mender instances:
+
+![region selector](region-selector.png)
```
Comment (tranchitella) [inline 10.General/00.Hosted Mender regions/docs.md:25]:
```suggestion
![region selector](region-selection.png)
```

**Comment by tranchitella** [comment]
@MaciejTe there are several other places in the docs where hosted.mender.io is mentioned; please add the note about the different regions everywhere.

File: 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md (hunk 1 above)
Comment (MaciejTe) [reply 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md]: done!

File: 10.General/00.Hosted Mender regions/docs.md (hunk 3 above)
Comment (MaciejTe) [reply 10.General/00.Hosted Mender regions/docs.md]: done!

File: 10.General/00.Hosted Mender regions/docs.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+
+## Switching between instances
+
+// TODO: is there an instruction about switching instances?
```
[the remark this answers is not in the document]
Comment (MaciejTe) [reply 10.General/00.Hosted Mender regions/docs.md]: done!

File: 10.General/00.Hosted Mender regions/docs.md (hunk 6 above)
Comment (MaciejTe) [reply 10.General/00.Hosted Mender regions/docs.md]: done!

File: 10.General/00.Hosted Mender regions/docs.md (hunk 4 above)
Comment (MaciejTe) [reply 10.General/00.Hosted Mender regions/docs.md]: done!

File: 10.General/00.Hosted Mender regions/docs.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+---
+title: Overview
+taxonomy:
+    category: docs
+---
+
+ender is available in multiple regions you can choose from, depending on your data processing and compliance needs. Each instance is independent of the others and provides all the Mender features.
```
Comment (tranchitella) [inline 10.General/00.Hosted Mender regions/docs.md:7]:
```suggestion
Hosted Mender is available in multiple regions you can choose from, depending on your data processing and compliance needs. Each instance is independent of the others and provides all the Mender features.
```

File: 10.General/00.Hosted Mender regions/docs.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+
+## Regions
+
+The list of the currently supported regions follows:
+
+* US instance - `https://hosted.mender.io`
+* EU instance - `https://eu.hosted.mender.io` 
+
+The EU instance is compliant with General Data Protection Regulation (GDPR).
```
Comment (tranchitella) [inline 10.General/00.Hosted Mender regions/docs.md:17]:
```suggestion
The EU instance is compliant with the General Data Protection Regulation (GDPR).
```

File: 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md [hunk 10]
```diff
@@ -181,6 +181,8 @@ Keep this terminal open as we will shortly use it to install the Mender client.
 
 ## Step 3 - Login to hosted Mender
 
+!!! Mender is available in multiple [regions](/10.General/00.Hosted%20Mender%20regions/docs.md) to connect to. Make sure you select your desired one before proceeding.
```
Comment (tranchitella) [inline 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md:184]: This applies to all the blocks:

```suggestion
!!! Hosted Mender is available in multiple [regions](/10.General/00.Hosted%20Mender%20regions/docs.md) to connect to. Make sure you select your desired one before proceeding.
```

**Review by tranchitella — COMMENTED** [review]
Sorry, another round of changes; after this we are okay, I believe.

File: 10.General/00.Hosted Mender regions/docs.md (hunk 8 above)
Comment (MaciejTe) [reply 10.General/00.Hosted Mender regions/docs.md]: Done, thanks!

File: 10.General/00.Hosted Mender regions/docs.md (hunk 9 above)
Comment (MaciejTe) [reply 10.General/00.Hosted Mender regions/docs.md]: Done, thanks!

File: 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md (hunk 10 above)
Comment (MaciejTe) [reply 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md]: Done, thanks!

**Review by tranchitella — APPROVED** [review]

**Pull request merged**